docs/testing/power-automate-troubleshooting.md

```diff
@@ -64,12 +64,27 @@ When building the **Run script** step of a Power Automate flow, you need to sele
 
 For more context on the Power Automate limitation and a discussion of potential workarounds for the dynamic selection of workbooks, see [this thread in the Microsoft Power Automate Community](https://powerusers.microsoft.com/t5/Power-Automate-Ideas/Allow-for-dynamic-quot-file-quot-value-for-excel-quot-get-a-row/idi-p/103091#).
 
+## Pass entire arrays as script parameters
+
+Power Automate allows users to pass arrays to connectors as a variable or as single elements in the array. The default is to pass single elements, which builds the array in the flow. For scripts or other connectors that take entire arrays as arguments, you need to press the **Switch to input entire array** button to pass the array an one complete object.
```

Comment:
It's implied that this button is the one outlined in red in the screenshot, but the icon is so ambiguous that I think the description should be a little more elaborate. Could we add something like "in the upper right corner of the..."? 
```suggestion
Power Automate allows users to pass arrays to connectors as a variable or as single elements in the array. The default is to pass single elements, which builds the array in the flow. For scripts or other connectors that take entire arrays as arguments, you need to select the **Switch to input entire array** button to pass the array as one complete object.
```

Reply:
I added "This button is in the upper-right corner of each array parameter input field." after the current text bock.